analyzer/src/main/kotlin/curation/Sw360PackageCurationProvider.kt

```diff
@@ -105,6 +105,7 @@ class Sw360PackageCurationProvider(sw360Configuration: Sw360StorageConfiguration
             config.password,
             config.clientId,
             config.clientPassword,
+            "",
```

Comment:
What is this new `token` parameter? I guess we need to make it configurable via `Sw360StorageConfiguration`?

Reply:
This parameter was introduced by one of the latest contributions to the client library. It allows specifying an access token directly rather than the parameters for obtaining the token from an OAuth server.

Yes, `Sw360StorageConfiguration` should be extended to support this parameter. When creating the connection to SW360 it has to be taken into account as well. I assume, this can be done in separate PRs.

Reply:
> It allows specifying an access token directly rather than the parameters for obtaining the token from an OAuth server.

So, does this mean the `token` parameter is mutually exclusive to the `username` / `password` pair?

Reply:
[Clarified](https://github.com/oss-review-toolkit/ort/pull/3588/files#diff-8fbb4de21ab99d226201dd7d6060d7395ee1b9698d0d2bc9c3f60ef0c290ca11R157-R158).